sdk/storage/azure-storage-blob/azure/storage/blob/_shared/models.py

```diff
@@ -307,11 +302,11 @@ def from_string(cls, string):
         :param str string: Specify service, container, or object in
             in the string with the first letter of the word.
         :return: A ResourceTypes object
-        :rtype: ~azure.storage.blob.ResourceTypes
+        :rtype: ~azure.storage.queue.ResourceTypes
```

Comment:
This probably needs to be adjusted back to blob here and the appropriate thing for each package. Same with some other return types in this file.